api/staged_products_property_collection_guid_assigner.go

```diff
@@ -115,6 +115,9 @@ func (item responsePropertyCollectionItem) getFieldValuesExceptGUID() map[interf
 		if key == "guid" {
 			continue
 		}
+		if !valueObj.(map[interface{}]interface{})["configurable"].(bool) {
```

Comment:
In terms of readability, the only think I would change is pulling out `!valueObj.(map[interface{}]interface{})["configurable"].(bool)` into
```
isNotConfigurable = valueObj.(map[interface{}]interface{})["configurable"].(bool)
if isNotConfigurable {
  continue
}
```

Reply:
Implemented.